ranger/tmpl/layout/authenticated_base.tmpl

```diff
@@ -14,7 +14,7 @@
     {{ template "additionalHeadScripts" }}
 
     {{ if not isDevelopment }}
-    {{ packTag "app" true }}
+    <link rel="stylesheet" href="{{asset "assets/style.css"}}">
```

Comment:
Nitpick: `<link rel="stylesheet" href="{{ asset "assets/style.css" }}">` for readability on the spacing around `{{}}`